test/server/api/redirect.test.ts

```diff
@@ -33,24 +36,97 @@ describe('redirect API tests', () => {
   beforeEach(() => {
     container
       .bind<CookieReducer>(DependencyIds.cookieReducer)
-      .to(CookieArrayReducerMock)
+      .to(CookieArrayReducerMockVisited)
   })
 
   afterEach(() => {
     container.unbindAll()
     loggerErrorSpy.mockClear()
   })
 
-  test('url exists in cache and db', async () => {
+  test('url exists in cache and db, real user unvisited', async () => {
```

Comment:
i notice a lot of bind/unbind going on. would snapshots help?

https://github.com/inversify/InversifyJS/blob/master/wiki/container_snapshots.md

Reply:
I opted not to use it as almost all tests in this file have a different combination of dependencies.